peeringdb_server/templates/site/view_exchange_bottom.html

```diff
@@ -87,10 +87,10 @@ <h5>{% trans "Prefixes" %}</h5>
      data-edit-target="api:ixfac">
   <div class="row">
     <div class="col-xs-6 col-sm-8 col-md-8">
-      <h5>Local Facilities</h5>
+      <h5>{% trans "Local Facilities" %}</h5>
     </div>
     <div class="col-xs-6 col-sm-4 col-md-4 filter">
-      <input type="text" placeholder="Filter" data-filter-target="#list-facilities" class="form-control" />
+      <input type="text" placeholder="{% trans "Filter" %}" data-filter-target="#list-facilities" class="form-control" />
```

Comment:
Same with placeholder, anything inside of the HTML tags should not be touched.

Reply:
Here I'm not sure, shouldn't that placeholder translation then also be removed?:
https://github.com/peeringdb/peeringdb/blob/06805fef36aed494d2729a132a702b25e997980b/peeringdb_server/templates/site/header.html#L54
That's the Advanced Search bar on the root page at beta.peeringdb.com and translated at the moment.

Reply:
@grizz I think he has a point here re placeholders.  The search one has been successfully localized, why not others?

Reply:
Err, yeah, my mistake.